images/installer/bin/README.md

```diff
@@ -0,0 +1,5 @@
+Utils for use during / after installation of image
+==================================================
+
+Contains image entrypoint as well as any other scripts useful
+in aiding an image run.
```

Comment:
I believe this file can be moved up one level and briefly document the contents, pointing to other readmes for details... it should make it easier to find and also avoid it ending up in `/usr/bin`